src/components/charts/BarChart.tsx

```diff
@@ -0,0 +1,191 @@
+import React from "react"
+import merge from "deepmerge"
+
+import {
+  ChartWrapper,
+  RecordSet,
+  RecordEntry,
+  ChartProps,
+} from "./ChartWrapper"
+
+interface BarChartProps extends ChartProps {
+  /**
+   * The order in which the x or y axis labels should be displayed.
+   */
+  xOrder?: string[]
+  /**
+   * WHether the bar chart should be horizontal.
+   */
+  horizontal?: boolean
+  /**
+   * Whether the bar chart should be stacked.
+   */
+  stacked?: boolean
+  /**
+   * The minimum value of the x (horizontal) or y axis.
+   */
+  min?: number
+  /**
+   * The maximum value of the x (horizontal) or y axis. Useful when data labels overflow.
+   */
+  max?: number
+  /**
+   * The width of the chart object. Defaults to 100% so it is responsive.
+   */
+  showLabels?: boolean
+}
+
+interface BarSeriesEntry {
+  name: string
+  data: (string | number)[]
+}
+
+export type BarSeries = BarSeriesEntry[]
+
+const makeDataLabelOptions = (
+  stacked: boolean,
+  horizontal: boolean,
+  min: number,
+  max: number
+) => {
+  const labelOptions: Record<string, unknown> = {}
+  if (stacked) {
```

Comment:
Should you have a check or at least a note that `stacked` and `horizontal` are not compatible options?

Reply:
First, it seems that `index.js` is what npm uses to structure packages. Not sure how to use it yet but will fill it out when I figure it out :D

Actually you can have a horizontal stacked bar chart. I have an example running in storybook :)